app/move/controllers/create/assessment.js

```diff
@@ -65,6 +66,11 @@ class AssessmentController extends CreateBaseController {
 
     super.saveValues(req, res, next)
   }
+
+  errorHandler(err, req, res, next) {
```

Comment:
We might want this at a more global level, like the parent form wizard class.

I think we need a way to handle the `prereq` step missing error in that parent class. And then maybe we just redirect to the start of the journey. I remember doing this elsewhere and will try and dig out the code for it.

Reply:
done - covered in https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/223